### DISCUSSION TITLE
Ruin rework feedback thread

### DISCUSSION THREAD

**Discussion by bananaman6213** [FakeFishGames/Barotrauma#14589] [Unstable feedback]
Making this as a place for people to add their own thoughts regarding the new ruins

I'll start:
Visually, everything looks great. The effects on the new turrets are especially nice, and fit the aesthetic. Having the slanted/curved walls in some rooms helps to break everything up a bit and make it look more distinct.

The new terminal mechanics are fun, and make you think for more than a moment if you didn't come in with trinkets. That said, there's not usually a reason to destroy a terminal when you could just leave the trinket in instead. The end effect is the same as far as I can tell, but you also get to pick the trinket back up on your way out. The only exception I've personally seen is the hatch in Alien_VaultModule2, which makes you burn a trinket (or spend ages crowbarring) to get at the artifact holder (which is a fine trade).

The idea that some rooms are supposed to be solved co-op (Alien_VaultModule1 and 2) is interesting, but I worry about singleplayer for these. You *can* force your way into them, but it isn't exactly fun.

My sample size isn't very high, but I feel like there aren't enough guardians in ruins. Other peoples' opinions on this would be much appreciated, but I found 3 guardians in two full ruin clears when testing. The ruins didn't feel very deadly, but that could be luck on my part. Again, more info needed.

Damage on the alien turret needs a buff. You'll likely be exploring ruins in at least a combat suit, which already has 25% deep tissue resistance. It took me over 20 seconds of sustained fire to get downed by one when testing. Even if you're a husk and not wearing a suit, the damage just isn't threatening.

**Comment by bananaman6213** [comment]
Adding individual module nitpicks here
![image](https://github.com/user-attachments/assets/2ae2d045-0c47-461b-a8f6-03a792b8e6f3)
One of the center symbols in Alien_VaultModule1 has an incorrect sprite depth and shows up above everything
![image](https://github.com/user-attachments/assets/ac4c615e-742b-40e5-99b1-8ca933f97d98)
Big decal in Alien_VaultModule2 obscures the "wire" leading to the button
![image](https://github.com/user-attachments/assets/444284d0-84c1-4cbf-a569-4b8e2661e0de)
The tunnel in this part of Alien_Chasm may be too wide, the turret likes to shoot past you if you're near the left side

The trap in Alien_Chasm2 is barely threatening, you can trigger all motion detectors while still being at a safe range

**Comment by ubertpendragon** [comment]
Ruins look great
Unbreakable walls are also good
New alien terminal behavior is much more interesting
Other unused items look promising


But:

- Rooms still are impossible to navigate for guardians. They will get stuck and wont be able to travel across rooms

![image](https://github.com/user-attachments/assets/44a3b56e-4c78-472e-9d46-79f4313552d4)
![image](https://github.com/user-attachments/assets/e89c8bd7-00d2-49e5-bc72-d8a64b41e5ca)
![image](https://github.com/user-attachments/assets/34c67970-df95-45b5-b6db-d13b2d41bd5f)

- In vault rooms, you can get stuck if the person who opened the door goes unconscious
Maybe add a button inside the vaults to open them?



- Some more extra outside decoration in the Z axis?
Ruins can feel like they float in the middle of nowhere

![image](https://github.com/user-attachments/assets/b679e37c-f1d8-4a76-9f2c-e4e8181bcdb3)
Tried adding some of the corner squares as background deco
![image](https://github.com/user-attachments/assets/bdce4630-0ee9-4d53-98db-550d2b34d861)

**Comment by bananaman6213** [comment]
![image](https://github.com/user-attachments/assets/67f63f04-2952-487a-8784-69c97162c0fb)
These two relays in Alien_VaultModule1 are not hidden in game
![image](https://github.com/user-attachments/assets/46f21cb8-072d-4403-b77a-50727377392d)
Alien_Shrine - This terminal, as far as i can tell, only exists to deactivate the motion detectors outside (and thus, the turret). Why would I use this when taking out the power cell from the generator accomplishes the same thing?
On a side note this module rocks, love it

Also, I've noticed it's very difficult to find weakened walls ingame. If I didn't already know they existed I'm not sure I'd ever notice them.

**Comment by bananaman6213** [comment]
More stuff, starting with a few wires that aren't hidden in game:
A couple in Alien_Chasm
![image](https://github.com/user-attachments/assets/d59b6d23-1953-446c-91cb-ca7ec048ac62)
One in Alien_Chasm2
![image](https://github.com/user-attachments/assets/3b08b6dd-80bf-44f2-bcf3-d5661bc5a8e2)

Wires coming off the timer circuits in Alien_MaintenancePassages, AlienOffices1, Alien_VaultModule2
![image](https://github.com/user-attachments/assets/d82a288e-fd3c-4d6b-8b3e-8df2d717a37e)
![image](https://github.com/user-attachments/assets/6c5bf32e-1651-4c5e-8e60-581c9e6eba02)
![image](https://github.com/user-attachments/assets/0102cc25-6651-4cb0-97e7-97e8d825117f)

This one's big. Alien_Storage1R has the trap of Alien_Chasm2 floating below it.
![image](https://github.com/user-attachments/assets/934316c4-95e3-42c6-add3-90aa0f76b4f6)

The timers in some modules are neat, though the wiring of them is slightly flawed. When spammed the lights get all jumbled and the hatch can start rapidly closing and opening. This can be fixed by selecting all delay components and setting them to reset when signal received. Also the lights are initially lit but after cycling once their resting state is with all lights off, which is odd. Could just invert all the light signals so that the whole timer basically ticks from on->off to off->on. Unsure if you even want/care about that though, and the former issue is bigger than the latter.

https://github.com/user-attachments/assets/920e8d4d-fe2b-4bec-a8e1-c225ada61d94
